images/gcp/centos-7.yaml

```diff
@@ -0,0 +1,20 @@
+---
+download_images: true
+
+packer:
+  # The project ID that will be used to launch instances and store images.
+  project_id: "eng-ksphere-platform"
+  # The source image to use to create the new image from.
+  source_image: "centos-7-v20220519"
+  # The source image family to use to create the new image from.
+  source_image_family: "centos-7"
```

Comment:
Please specify 7.9 and not just centos-7

Reply:
Im not sure how to do so for gcp, but will search for it.

Reply:
FYI: CentOS Linux release 7.9.2009 (Core)